openapi_client_generator/python/model.mustache

```diff
@@ -99,8 +99,10 @@ class {{classname}}(object):
         :type: {{datatype}}
         """
 {{#required}}
+{{^nullable}}
         if self._configuration.client_side_validation and {{name}} is None:
             raise ValueError("Invalid value for `{{name}}`, must not be `None`")  # noqa: E501
+{{/nullable}}
 {{/required}}
 {{#isEnum}}
 {{#isContainer}}
```

Comment:
Ok do we have to change this everytime we generate the API code? In that case I would make an additional change in the `generate_openapi_client.sh` script which applies this as a patch.